helm/k8s-deployment-scaler/templates/role.yaml

```diff
@@ -0,0 +1,9 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: Role
+metadata:
+  name: k8s-deployment-scaler-role
+  namespace: default
```

Comment:
There is a bit of inconsistency between resource definitions, some of them specify namespace and other `default`.

In any case, `default` namespace doesn't seem like the best place to run kubernetes controller like this.

Reply:
role.yaml has been removed, it became YAGNI